### PR TITLE
Config ui workflow blog post

### PR DESCRIPTION
**Pull request by bleonard**
screencapture-localhost-3001-blog-software-development-workflow-2021-07-19-17_22_56

### CODE REVIEW

File: pages/blog/software-development-workflow.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Software Development Workflow with the Grouparoo UI"
```
Comment (teallarson) [inline pages/blog/software-development-workflow.mdx:2]: Per our SEO suggesting <60 chars for page titles, we may want something like "Development Workflow with the Grouparoo UI" instead (58 characters once we add ": Grouparoo Blog")

File: pages/blog/software-development-workflow.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+As this data got more and more useful, other teams wanted to be able to put it into action. In response to that need, we work with companies that are looking to operationalize their data warehouse. Data teams are calling this [Reverse ETL](https://www.grouparoo.com/solutions/reverse-etl) because it is writing data back to the tools the business uses.
+
+These integrations create more leverage, but the worst case scenario has escalated. A [Marketo data integration](https://www.grouparoo.com/integrations/destinations/marketo) could send a million emails to the wrong people. A more rigorous workflow is needed.
+
+## Grouparoo Process
+
+In Grouparoo, our JSON-based config files map the data from your source data warehouse to destinations like [Salesforce](https://www.grouparoo.com/integrations/destinations/salesforce), [Zendesk](https://www.grouparoo.com/integrations/destinations/zendesk), and [others](https://www.grouparoo.com/integrations).
```
Comment (teallarson) [inline pages/blog/software-development-workflow.mdx:52]: SEO might be improved with "like our Salesforce Data Integration, our Zendesk integration, and others"

**Review by teallarson — COMMENTED** [review]
Exciting!  Two notes for SEO purposes.

File: pages/blog/software-development-workflow.mdx (hunk 1 above)
Comment (andyjih) [reply pages/blog/software-development-workflow.mdx:2]: "Development workflow for Reverse ETL" is probably my favorite.

**Pull request merged**